update july - rules (#694)
akeneo/pim-helpcenter: content/updates/2020-07/1-rules-UI.md

---
pim_announcement_img: ./img/rules_UI.png
pim_announcement_alt_img: "The UI to manage the rules"
pim_announcement_audience:
- EE
---

> # A brand new UI to manage your rules

Reviewer: ```suggestion
# New! Create and update rules from the PIM!
```
I'm trying the "!" here ;)

Reviewer: Il manque les meta data non ?

Reviewer: @morganen si toutes les actions/opérations ne sont pas encore dispo il faut peut-être le dire ?

Author: J'hésitais justement, je me demandais si ça ne faisait pas trop "négatif" de le dire ici, sachant que je le dis déjà dans la doc à plein de reprises. Mais oui c'est sûrement utile de le redire ici

Author: meta data rajoutées, sorry pour l'oubli.

> In order to let everyone use the rules engine and boost your productivity, we have created a user interface to manage your rules directly in the PIM.  

Reviewer: ```suggestion
To ease the use of the rules engine and boost your productivity, we created a brand new user interface to directly manage your rules in the PIM.  
```

> We know that your time is precious and that you always need to work efficiently. That's why we built the rules engine a few years ago.

Reviewer: ```suggestion
Our rules engine enables you to automate enrichment actions on specific product selections. This new screen allows you to create, update and delete rules in a few clicks, it will definitely ease and faster the rules management. 
The imports of YML files to create or update rules remains available.
```

> Indeed, this feature allows you to automate actions on a selection of products/product models. But until now, the only way to create/update rules was via a YML file import, which was not simple enough for our non-technical users.  

Reviewer: ```suggestion
```

Reviewer: I wouldn't insist on the fact that the YML files are not easy to use for non tech users.

> Thanks to this brand new user interface, you'll be able to create/update your rules with only some clicks :wink:.

Reviewer: ```suggestion
```

---
pim_announcement_img: ./img/rules_UI.png
pim_announcement_alt_img: "The UI to manage the rules"
pim_announcement_audience:
- EE
---

# New! Create and update rules from the PIM!

> ::: meta-data type="New" features="Rules engine, Productivity" available="early July" link-to-doc="../articles/manage-your-rules.html#the-rule-builder-tab.html"

Reviewer 2: Isn't it "mid-July" instead of early July, @morganen?

Author: indeed. It was the 16th

:::

To ease the use of the rules engine and boost your productivity, we created a brand new user interface to directly manage your rules in the PIM.  

> 

Reviewer 2: It's missing the image of the feature directly in the article, isn't it?

---
pim_announcement_img: ./img/Rules_UI.png
pim_announcement_alt_img: "The UI to manage the rules"
pim_announcement_audience:
- EE
---

# New! Create and update rules from the PIM!
::: meta-data type="New" features="Rules engine, Productivity" available="mid-July" link-to-doc="../articles/manage-your-rules.html#the-rule-builder-tab.html"
:::

To ease the use of the rules engine and boost your productivity, we created a brand new user interface to directly manage your rules in the PIM.  

![New screen to manage your rules](../img/Rules_UI.png)

> Our rules engine enables you to automate enrichment actions on specific product selections. This new screen allows you to create, update and delete rules in a few clicks, it will definitely ease and faster the rules management.

Reviewer 2: ```suggestion
Our rules engine enables you to automate enrichment actions on specific product selections. This new screen allows you to create, update and delete rules in a few clicks, it will definitely ease and faster the rules management.  
```
